cmd/storage-datatypes.go

```diff
@@ -112,6 +112,13 @@ type FileInfo struct {
 	// a deleted marker for a versioned bucket.
 	Deleted bool
 
+	// Transitioned is set when this FileInfo represents
+	// a transitioned entry for an object version.
+	Transitioned bool
+	// TransitionStatus is set to Pending/Complete for transitioned
+	// entries based on state of transition
+	TransitionStatus string
```

Comment:
Why is the boolean needed? Is it possible to just use the string?

Reply:
with transition queuing, this is no longer needed. Removed it